src/Core/Components/AppBar/FluentAppBarItem.razor.cs

```diff
@@ -54,6 +54,12 @@ public partial class FluentAppBarItem : FluentComponentBase, IDisposable
     [Parameter]
     public int Count { get; set; } = 0;
 
+    /// <summary>
+    /// Gets or sets the position of the item in the app bar.
+    /// </summary>
+    [Parameter]
+    public int Order { get; set; }
```

Comment:
Isn't it possible to use the order of the list elements?
It's like the user writing `<li>` elements, the order defined will be the order of the result.

Reply:
After checking the example, I think this property should not be included in the interface. It's the user who chooses to order the elements using one criterion or another.

Reply:
Yes, they can use the order of the list elements (by just using `Items="the _list"`)

The `Order` parameter is meant to explicitly set a custom order. I'll remove it from the interface as it can easily be  added when creating a class derived from `FluentAppBarItem`. I'll adjust the sample to that